presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -171,6 +179,11 @@ protected void launchPrestoCliWithServerArgument(String... arguments)
                 prestoClientOptions.add("--krb5-disable-remote-service-hostname-canonicalization");
             }
         }
+
+        if (ldapAuthentication) {
```

Comment:
Implementing it with echo is not convenient. We need to hack tempto, we need to hack `run_on_docker` script. This PR has a lot of dependencies. It depends on ldap authentication, on ssl in opensource jdbc, on ssl internal communication. So, it is long way until it will land on the master. Hacking around the stuffs is painful, because you will need to rebase all that changes all the way.

I believe that having hidden parameter for password is fine. All the cli clients (mysql, mssql, hive, oracle, postgree) have so. We can try to convince David to merge it. Because it is really convenient for testing. If he insisted  - then we would hack around. I just don't want to run ahead of time.